src/platforms/esp32/components/avm_sys/platform_nifs.c

```diff
@@ -450,8 +454,38 @@ static term nif_atomvm_platform(Context *ctx, int argc, term argv[])
     return ESP32_ATOM;
 }
 
+static term nif_esp_get_mac(Context *ctx, int argc, term argv[])
+{
+    UNUSED(argc);
+
+    GlobalContext *global = ctx->global;
+
+    esp_mac_type_t interface;
+    if (globalcontext_is_atom_index_equal_to_atom_string(global, argv[0], ATOM_STR("\x8", "wifi_sta"))) {
+        interface = ESP_MAC_WIFI_STA;
+    } else if (globalcontext_is_atom_index_equal_to_atom_string(global, argv[0], ATOM_STR("\xB", "wifi_softap"))) {
+        interface = ESP_MAC_WIFI_SOFTAP;
+    } else {
+        RAISE_ERROR(BADARG_ATOM);
+    }
```

Comment:
I think that we might define something similar to this:
```c
 91 static const AtomStringIntPair cmd_table[] = {
 92     { ATOM_STR("\x4", "read"), UARTReadCmd },
 93     { ATOM_STR("\x5", "write"), UARTWriteCmd },
 94     { ATOM_STR("\x5", "close"), UARTCloseCmd },
 95     SELECT_INT_DEFAULT(UARTInvalidCmd)
 96 };
```

And then use `interop_atom_term_select_int(cmd_table, cmd_term, ctx->global);`

Reply:
done.